module/input/SensorFilter/src/SensorFilter.hpp

```diff
@@ -42,12 +43,13 @@ using extension::Configuration;
 
 namespace module::input {
 
-    using message::actuation::KinematicsModel;
     using message::behaviour::state::Stability;
     using message::behaviour::state::WalkState;
     using message::input::Sensors;
     using message::platform::RawSensors;
 
+    using namespace tinyrobotics;
```

Comment:
hmmm do we want to do this

Reply:
Hmm, probably best to not tbh. Will update.